integrations/sbt-bloop/src/main/scala/bloop/integrations/sbt/SbtBloop.scala

```diff
@@ -73,12 +73,23 @@ object PluginImplementation {
       val aggregates = project.aggregate.map(agg => makeName(agg.project, configuration))
       val dependenciesAndAggregates = dependencies ++ aggregates
 
+      val bloopConfigDir = BloopKeys.bloopConfigDir.value
       val scalaName = "scala-compiler"
       val scalaVersion = Keys.scalaVersion.value
       val scalaOrg = Keys.ivyScala.value.map(_.scalaOrganization).getOrElse("org.scala-lang")
       val allScalaJars = Keys.scalaInstance.value.allJars.map(_.getAbsoluteFile)
       val classpath = PluginDefaults.emulateDependencyClasspath.value.map(_.getAbsoluteFile)
-      val classesDir = Keys.classDirectory.value.getAbsoluteFile
+      val bloopTarget = Defaults.makeCrossTarget(
+        bloopConfigDir,
+        Keys.scalaBinaryVersion.value,
+        (Keys.sbtBinaryVersion in Keys.pluginCrossBuild).value,
+        Keys.sbtPlugin.value,
+        Keys.crossPaths.value
+      )
+
+      val classesDir = bloopTarget / (Defaults.prefix(configuration.name) + "classes")
```

Comment:
Will the classes directories live in `.bloop/` or in some nested directory? I would like to be able to easily remove all `classes` directories without removing `.bloop/*.config` files

Reply:
`.bloop/target...` etc. It will have the same path as it used to have in `target/` (sbt) but this time rebased to `.bloop`.

Reply:
I would consider specifying the location per project, it's good to know when you want to `rm -rf` and I should be able to rely on this location not changing.

Reply:
transparent cache locations are a nice and cheap feature to support